modules/ROOT/nav.adoc

```diff
@@ -14,7 +14,10 @@
  ** xref:configuring-runtime-fabric.adoc[Configuring Runtime Fabric]
   *** xref:associate-environments.adoc[Associating Business Groups and Environments]
   *** xref:install-rtfctl.adoc#configure-rtfctl[Configuring Runtime Fabric Using RTFCTL]
-  *** xref:custom-ingress-configuration.adoc[Configuring Ingress]
+  *** xref:configure-ingress.adoc[Configuring Ingress]
+  **** xref:custom-ingress-configuration.adoc[Configuring Ingress Resource Template]
+  **** xref:configure-ingress-http-resource.adoc[Configuring Ingress with HTTP Route Template Custom Resource In Runtime Fabric]
+  **** xref:configure-ingress-endpoints.adoc[Configuring Ingress Endpoints for Mule App Deployment]
```

Comment:
Do you need these hard-coded link texts?

Reply:
(Please don't resolve comments without either taking action or saying why you didn't change - we need input from writers on which comments are off base...)